src/h3lib/lib/h3Index.c

```diff
@@ -879,18 +881,18 @@ int H3_EXPORT(maxFaceCount)(H3Index h3) {
  * @param h3 The H3 index
  * @param out Output array. Must be of size maxFaceCount(h3).
  */
-void H3_EXPORT(h3GetFaces)(H3Index h3, int* out) {
+void H3_EXPORT(getIcosahedronFaces)(H3Index h3, int* out) {
     int res = H3_GET_RESOLUTION(h3);
-    int isPentagon = H3_EXPORT(h3IsPentagon)(h3);
+    int pentagon = H3_EXPORT(isPentagon)(h3);
```

Comment:
Again, I don't really like `pentagon` here - maybe `isCellPentagon`? I wish there was a nicer rule here.

Reply:
I'd rather not change this one, though.

Reply:
As above, maybe `int isPent = H3_EXPORT(isPentagon)(h3);`?